docs/project/2024/alberto.udrea0412/index.md

```diff
@@ -0,0 +1,78 @@
+# Weather Station
+
```

Comment:
Please add a one line description of your project.

Reply:
In the description section (i.e. shorten the existing) or make a new shorter description right below the title?

Reply:
A shorter description under the title.